docs/blog.md

```diff
@@ -0,0 +1,18 @@
+Am Anfang war ich richtig verloren muss ich sagen, doch zu meiner Verteidigung habe ich seit einem Jahr nicht mehr programmiert habe. 
+Die ersten paar Zeilen waren recht harzig, doch desto weiter ich kam umso mehr spass hatte ich. 
+Dass fande ich meinen Lustigsten Fehler, doch ich finde ein Stern sah besser aus als der der Verlangt war.
+
+from turtle import *
+fillcolor("red")
+begin_fill()
+for x in range(6):
+    forward(100)
+    left(140)
+    backwardward(100)
+    right(80)
```

Comment:
````suggestion
```py
from turtle import *
fillcolor("red")
begin_fill()
for x in range(6):
    forward(100)
    left(140)
    backwardward(100)
    right(80)
```
````

--> Python Code in Codeblock